packages/file/package.json

```diff
@@ -19,10 +19,12 @@
     "test": "jest",
     "format-check": "prettier --check **.{ts,md} --ignore-path ../../.prettierignore"
   },
-  "devDependencies": {
-    "@sketch-hq/sketch-file-format-ts": "6.0.2",
-    "@types/adm-zip": "0.4.34",
+  "dependencies": {
     "adm-zip": "0.5.5",
-    "node-stream-zip": "1.13.3"
+    "node-stream-zip": "1.13.3",
+    "@sketch-hq/sketch-file-format-ts": "6.0.2"
+  },
+  "devDependencies": {
+    "@types/adm-zip": "0.4.34"
```

Comment:
Shouldn’t this be in `dependencies` and `sketch-file-format-ts` down here in `devDependencies`?

Reply:
`adm-zip`, `node-stream-zip` and `@sketch-hq/sketch-file-format-ts` are required by the actual implementation:

https://github.com/sketch-hq/sketch-document/blob/6ae901db0a5b4da29340fdc3d894bb4dfbb57aad/packages/file/src/file.ts#L2-L4

My understanding is they must therefor be declared as `dependencies` so that they get installed with `sketch-file`.

However, I do wonder if `@sketch-hq/sketch-file-format-ts` is different because it is "just" the type, not an API as such. Maybe @jedrichards can help shed some light on it.

Reply:
In any case, `adm-zip` should be in `dependencies`, not `devDependencies` (right? It’s Thursday, I could never get the hang of Thursdays)

Reply:
`adm-zip` yes but `@types/adm-zip` is not required by the production bundle, it's merely there for convenience during development.

Reply:
Umm, I think the answer is it _depends_.

Types can sometimes contain actual implementation JS as well as "just" types, namely when enums are involved. E.g. checkout the compiled JS for `sketch-file-format-ts`, there's implementation in there that could potentially be in use:

https://unpkg.com/browse/@sketch-hq/sketch-file-format-ts@6.0.2/dist/cjs/types.js

So I'd say the safest thing is to keep these types in `dependencies`.

I think there are cases where you might just be using types from a 3rd package purely in the build phase, and you never end up exposing any part of them in your package API, in which case it'd be fine to put them in `devDependencies` I think.

Reply:
Okay, great. Thank you @jedrichards. So just to confirm, the implementation does not need further changes?

Reply:
Sorry about the noise, I totally missed the `@types/` bit in front of `adm-zip` and it does indeed belong in `devDependencies` then.